### ISSUE TITLE
Harmonize bar/pub icons

### ISSUE THREAD

**Issue by jla217** [organicmaps/organicmaps#9470]
Since the September release, there's a new pub icon, and the bar icon has changed. The discrepancy between map and bookmark rendering could however be improved.

_Current behaviour (see screenshots):_
- Bar: cocktail glass on map, beer pint on bookmark list.
- Pub: beer pint on map, pin with arrow on bookmark list.

_Suggested enhancement:_ Cocktail glass for bar, beer pint for pub on both map and bookmark list.

![photo_5776018985307981303_y](https://github.com/user-attachments/assets/9eee528d-3ccd-4630-af74-5b7ec3d16d16)
![photo_5776018985307981302_y](https://github.com/user-attachments/assets/f359738a-4a04-4cbe-a3fb-05ada0e3aadf)

**Comment by biodranik** [comment]
@dvdmrtnz @strump is it related to the recent icon hotfix?

**Comment by dvdmrtnz** [comment]
> @dvdmrtnz @strump is it related to the recent icon hotfix?

No. We have changed the bookmark icons on the map to match the POI icons on the map.

However, icons on the bookmark lists do not match with the icons on the map. These have not been changed. They should be updated.

**Comment by strump** [comment]
Looks like android resources and map icons have differences:

| File | Image |
| ----| ---- |
| `android/app/src/main/res/drawable/ic_bookmark_bar.xml` | <img width="140" alt="Screenshot 2024-10-10 at 11 49 41" src="https://github.com/user-attachments/assets/1a75ca72-945d-456d-979d-a9c2caa9fdf8"/> |
| `data/styles/default/light/symbols/pub-m.svg` |  <img width="140" alt="Screenshot 2024-10-10 at 11 51 08" src="https://github.com/user-attachments/assets/34dedd68-d53f-4f72-9053-736beb1462d5"/> |

**Comment by dvdmrtnz** [comment]
iOS icons also need to be updated:

<img src="https://github.com/user-attachments/assets/3fe3caba-d3a7-42d7-b8c4-901b6ace9b46" width="320"/> <img src="https://github.com/user-attachments/assets/c5df986c-55cb-4f61-9340-6977cef085a7" width="320"/>

**Comment by dvdmrtnz** [comment]
This has been fixed in https://github.com/organicmaps/organicmaps/pull/10953
